### PR TITLE
Display two locations on map

### PR DESCRIPTION
**Pull request by krystalwhite**
This code accomplishes a few things:
1. Updated the javascript file to display a starting and ending location with a line between them
2. Updated stylings with the fragments
3. Deleted the style.css file and copied over the #map text to styles.css
4. Organized map information into its own template folder
5. Modified controller to send to mapDisplay.html

This code does NOT take in typed location inputs, but instead provides selections from drop-down menus. The code will need to be refactored to take in typed locations at a future point.

### CODE REVIEW

File: src/main/resources/static/styles.css [hunk 1]
```diff
@@ -97,4 +97,29 @@ color: red;
 footer{
     margin-top: auto;
     text-align: center;
+}
+
+
```
Comment (miketully426) [inline src/main/resources/static/styles.css]: Only one space between blocks needed

File: src/main/resources/static/styles.css [hunk 2]
```diff
@@ -97,4 +97,29 @@ color: red;
...
+#map {
+ height: 100%;
+}
+html,
```
Comment (miketully426) [inline src/main/resources/static/styles.css:102]: At least one space between blocks needed

File: src/main/resources/static/script.js [hunk 3]
```diff
@@ -1,11 +1,35 @@
...
-
 function initMap() {
-  map = new google.maps.Map(document.getElementById("map"), {
-    center: { lat: -34.397, lng: 150.644 },
-    zoom: 8,
+  const directionsService = new google.maps.DirectionsService();
+  const directionsRenderer = new google.maps.DirectionsRenderer();
+  const map = new google.maps.Map(document.getElementById("map"), {
+    zoom: 7,
+    center: { lat: 41.85, lng: -87.65 },
```
Comment (miketully426) [inline src/main/resources/static/script.js]: I'd suggest pulling these starting lat-long coordinates and zoom out and setting them as global variables at the start of the file. This way they aren't "magic numbers" in the code and can easily be found to edit in the future.

**Review by miketully426 — CHANGES_REQUESTED** [review]

**Review by miketully426 — COMMENTED** [review]
Much better! Good job!

**Review by emmykg38 — APPROVED** [review]
Code looks clean (Mike has a meticulous eye I am trying to acquire...)
Nice work, Krystal.

**Review by mgunnhawkins — APPROVED** [review]

**Review by miketully426 — APPROVED** [review]

**Pull request merged**